test/models/RegExBuilder/Model.test.ts

```diff
@@ -0,0 +1,22 @@
+import { RegExBuilder } from "$models/RegExBuilder";
+
+describe("RegExBuilder", () => {
+  it("validates different phone numbers formats", () => {
+    expect("12345678").toMatch(RegExBuilder.phoneNumber());
+    expect("01123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("1123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("541123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("5491123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("+5491123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("91123456789").toMatch(RegExBuilder.phoneNumber());
+    expect("+19142329901").toMatch(RegExBuilder.phoneNumber());
+    expect("+442087599036").toMatch(RegExBuilder.phoneNumber());
+  });
+
+  it("validates invalid phone numbers formats for negative output", () => {
```

Comment:
Y aca tmb, haria un `it` por cada uno, y en el mensaje de ir pondria por que daria falso, algo como:
```typescript
it("returns false if the value contains a chacacter", () => {
   expect("a12345678".match(RegExBuilder.phoneNumber())).toBe(false);
});
```